### ISSUE TITLE
Regular expression error in FENCED_BLOCK_RE

### ISSUE THREAD

**Issue by a-guzhin** [mkdocs-plugins/termynal#16]
The current regular expression doesn't match Termynal code blocks correctly.

For example:
````yaml
 Welcome

One:

```yaml title="files/one.yml"
--8<-- "files/one.yml"
```

Two:

```yaml title="files/two.yml"
--8<-- "files/two.yml"
```

<!-- termynal -->

```
$ command

Ok!
```
````

Two blocks are selected when only one should be:
````yaml
```

Two:

```yaml title="files/two.yml"
--8<-- "files/two.yml"
```
````

````yaml
```
$ command

Ok!
```
````

This leads to errors, for example, when using the pymdownx.snippets extension (--8<-- notation).
[example.zip](https://github.com/mkdocs-plugins/termynal/files/12774186/example.zip)

**Comment by daxartio** [comment]
Fixed in 0.11.1

**Comment by daxartio** [comment]
It broke

<img width="799" alt="image" src="https://github.com/mkdocs-plugins/termynal/assets/11211587/938afc8e-1599-48a2-bb27-f603179222b7">

There was code as an example that showed how it could be used
